app/services/utility_service/south/response_mapper.rb

```diff
@@ -20,6 +24,27 @@ def map_books(books)
           }
         end
       end
+
+      def map_notes(notes)
+        notes.map do |note|
+          {
+            title: note['TituloNota'],
+            type: note['ReseniaNota'] ? :review.to_s : :critique.to_s,
+            created_at: note['FechaCreacionNota'],
```

Comment:
Por qué no directamente devolver el string?
```suggestion
            title: note['TituloNota'],
            type: note['ReseniaNota'] ? 'review' : 'critique',
            created_at: note['FechaCreacionNota'],
```